sync/vm/registry.go

```diff
@@ -5,12 +5,24 @@ package vm
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"reflect"
+	"sync"
+	"sync/atomic"
 
 	synccommon "github.com/ava-labs/coreth/sync"
 	"github.com/ava-labs/libevm/log"
 )
 
+var (
+	errClientCannotProvideSummary = errors.New("client cannot provide a summary")
```

Comment:
```suggestion
	errClientCannotProvideSummary = errors.New("client must provide a summary")
```
I think this will ultimately be whatever the most recent `target` is, but a sync summary is ok for now